src/pages/reset-password/index.js

```diff
@@ -13,8 +13,9 @@ import Login from 'common/login'
 
 const StyledContainer = styled(Container)`
     text-align: center;
-    height: 72rem;
+    height: 100vh;
```

Comment:
in mobile, most probably the user need to scroll, if it is 100vh im afraid they wont be able to scroll

Reply:
Seems alright when I tested in for iPhone 5/SE 🙏